grafana_client/elements/service_account.py

```diff
@@ -0,0 +1,114 @@
+from .base import Base
+
+
+class ServiceAccounts(Base):
+    def __init__(self, client):
+        super(ServiceAccounts, self).__init__(client)
+        self.client = client
+
+    def search_service_accounts(self, query=None, page=1, perpage=None):
+        """
+
+        :return:
+        """
+        list_of_sa = []
+        sa_on_page = None
+        show_sa_path = "/serviceaccounts/search"
+        params = []
+        if query:
+            params.append("query=%s" % query)
+
+        if page:
+            iterate = False
+            params.append("page=%s" % page)
+        else:
+            iterate = True
+            params.append("page=%s")
```

Comment:
This is probably not intended if `page` is undefined?
```suggestion
```

Reply:
Hi Andreas, this was intended to be used when no page was given so it can iterate through the pages in order to search the service account.